archetypes/default.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+date: {{ .Date }}
+draft: true
+---
+
+## Summary
+
+_What is needed. A one paragraph explanation of what this is about._
+
+## Motivation
+
+_Why is it needed? What does it unlock?_
+
+_Do any practical workarounds exist? What is the current pattern or hack to achieve a similar outcome._
+
+## Usage Documentation
+
+_Document new features as if they are already implemented. Provide code examples using the new api. This section should introduce new concepts and terms, describe use cases or situations._
+
+## Status
```

Comment:
At first glance this section looked like a duplication of [`/data/topics.json`](https://github.com/ipfs-shipyard/arewedistributedyet/blob/master/data/topics.json), but think I get it now. 
This will be extensive elaboration, while the landing page will have a short summary – is that the intention?

Reply:
Yes, that's what I'm thinking, this is the "formal proposal" that gets published under the "read more" link on the home page.